packages/babel-plugins/@glimmer/babel-plugin-strict-template-precompile/index.js

```diff
@@ -1,4 +1,7 @@
 const { precompile } = require('@glimmer/compiler');
+const { parse } = require('@babel/parser');
+const { default: generate } = require('@babel/generator');
```

Comment:
p1/5. 

I _vaguely_ prefer this (I think it is easier to parse at a glance):

```suggestion
const generate = require('@babel/generator').default;
```

Reply:
I actually tried to do that, but it wasn't working, said `generate` was not a function. I think it might be due to the way they write/transpile it for publishing.